cf-deployment.yml

```diff
@@ -305,6 +305,7 @@ instance_groups:
             override: true
             refresh-token-validity: 2592000
             scope: cloud_controller.read,cloud_controller.write,openid,password.write,cloud_controller.admin,scim.read,scim.write,doppler.firehose,uaa.user,routing.router_groups.read,routing.router_groups.write
+            secret: ''
```

Comment:
@fhanik Makes sense not to have a default for password. Should `secret` have a real (i.e. securely and randomly generated) value instead of an empty string? What effect would that have?